src/nvim/charset.c

```diff
@@ -244,18 +244,18 @@ int buf_init_chartab(buf_T *buf, int global)
                  || (p_altkeymap && (F_isalpha(c) || F_isdigit(c))))
                 && !(enc_dbcs && (MB_BYTE2LEN(c) == 2))) {
               if (tilde) {
-                chartab[c] = (chartab[c] & ~CT_CELL_MASK)
-                             + ((dy_flags & DY_UHEX) ? 4 : 2);
-                chartab[c] &= ~CT_PRINT_CHAR;
+                chartab[c] = (uint8_t)((chartab[c] & ~CT_CELL_MASK)
+                                      + ((dy_flags & DY_UHEX) ? 4 : 2));
```

Comment:
Right, I corrected this one earlier... Something must have gone wrong with one of my rebase. Sorry about that. Going to fix it now